docker-compose.yaml

```diff
@@ -1,9 +1,18 @@
-version: '3'
+version: '3.4'
```

Comment:
will this affect anything critical in the existing users' deployments? Should we account for sth or notify them that they will need to download sth new?

Reply:
3.4 is released 2+ years ago so the users should already have the updated version (because of security issues as well).

Saying that, we can have a message asking for updating the docker-compose version in case the have an earlier one.